### PR TITLE
feat(accessibility): add the accessibility status of the website

### LINKED ISSUE #3404: Afficher notre niveau d'accessibilité sur toutes les pages + une page explicative
# Rappel des obligations 

En tant que site public, nous avons l'obligation même si n'avons pas effectué d'audit d'accessibilité d'afficher notre niveau d'accessibilité. 
Tant que le niveau de conformité est inconnu, il est considéré comme non conforme.

cf.image issue du site https://pidila.gitlab.io/checklist-pidila/
Capture d’écran 2021-03-31 à 17 21 34

Autre source d'explication => https://www.numerique.gouv.fr/publications/rgaa-accessibilite/obligations/#mention-obligatoire-sur-la-page-daccueil

Autre obligation cette mention de notre niveau d'accessibilité doit ensuite être un lien vers une page dédiée à la déclaration d'accessibilité.
Capture d’écran 2021-03-31 à 17 20 15

# 1. Ajouter "Accessibilité : non conforme" dans le footer 

Voici un exemple d'affichage du niveau. 
Capture d’écran 2021-03-31 à 17 12 03

Ajouter "Accessibilité : non conforme" dans le footer dans la colonne après "Mentions légales" et avant "Politique de confidentialité"
Capture d’écran 2021-03-31 à 17 16 42

# 2. Une page dédiée à la déclaration d'accessibilité 

3 exemples de pages qui n'ont pas encore effectué d'audit d'accessibilité : 
- page de [g_nuis e sante](https://gnius.esante.gouv.fr/accessibilite)
- page de [Betagouv](https://beta.gouv.fr/accessibilite/) 

## Contenu 
Mettre un lien derrière "Accessibilité : non conforme" qui renvoie à une page comprenant ce texte => [ICI](https://docs.google.com/document/d/1ouk54P4pInmXkDrecuHclSemTh8ZrhSsH9_7Rqar9p4/edit#)

## Design 
Pour le design on reprend le même que pour les pages A propos, Mentions Légales ou Politique de confidentialité. 
Exemples : 
- https://cdtn-preprod-code-travail.dev2.fabrique.social.gouv.fr/a-propos
- https://cdtn-preprod-code-travail.dev2.fabrique.social.gouv.fr/politique-confidentialite

### PR DESCRIPTION
**Pull request by m-maillot**
Closes #3404

### CODE REVIEW

File: packages/code-du-travail-frontend/pages/accessibility.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+          <Title shift={theme.spacings.larger}>
+            Déclaration d’accessibilité
+          </Title>
+          <p>
+            Le Ministère du travail, de l’emploi et de l’insertion s’engage à
+            rendre son service accessible conformément à l’article 47 de la loi
+            n° 2005-102 du 11 février 2005.
+          </p>
+          <p>
+            A cette fin, il met en œuvre la stratégie et l’action suivante :
```
Comment (lionelB) [inline packages/code-du-travail-frontend/pages/accessibility.js]:
```suggestion
            À cette fin, il met en œuvre la stratégie et l’action suivante :
```

File: packages/code-du-travail-frontend/pages/accessibility.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          </p>
+          <Title shift={theme.spacings.larger}>Résultat des tests</Title>
+          <p>
+            L’audit de conformité est en attente de réalisation (milieu de
+            l’année 2021).
+          </p>
+          <Title shift={theme.spacings.larger}>
+            Établissement de cette déclaration d’accessibilité
+          </Title>
+          <p>Cette déclaration a été établie le 1 avril 2021.</p>
```
Comment (lionelB) [inline packages/code-du-travail-frontend/pages/accessibility.js]: :)

File: packages/code-du-travail-frontend/pages/accessibility.js [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+          <Title shift={theme.spacings.larger}>Amélioration et contact</Title>
+          <p>
+            Si vous n’arrivez pas à accéder à un contenu ou à un service, vous
+            pouvez contacter le responsable de Code.travail.gouv.fr pour être
```
Comment (lionelB) [inline packages/code-du-travail-frontend/pages/accessibility.js]:
```suggestion
            pouvez contacter le responsable du site Code du travail numérique pour être
```

File: packages/code-du-travail-frontend/src/layout/Footer.js [hunk 4]
```diff
@@ -83,6 +83,11 @@ const Footer = () => {
                       <StyledLink>Mentions légales</StyledLink>
                     </Link>
                   </StyledListItem>
+                  <StyledListItem>
+                    <Link passHref href="/accessibility">
```
Comment (lionelB) [inline packages/code-du-travail-frontend/src/layout/Footer.js]:
```suggestion
                    <Link passHref href="/accessibilité">
```

File: packages/code-du-travail-frontend/pages/accessibility.js (hunk 2 above)
Comment (m-maillot) [reply packages/code-du-travail-frontend/pages/accessibility.js]: Indeed, I did not pay attention to the date 😂

File: packages/code-du-travail-frontend/src/layout/Footer.js (hunk 4 above)
Comment (m-maillot) [reply packages/code-du-travail-frontend/src/layout/Footer.js]: Are we sure is it a good idea to put an accented character in the URL (and in the naming of a file)? Perhaps `/accessibilite`?

Comment (lionelB) [reply packages/code-du-travail-frontend/src/layout/Footer.js]: ok for accessibilite

**Comment by m-maillot** [comment]
Feedbacks taken into account :)

**Review by lionelB — APPROVED** [review]

**Pull request merged**